src/type.zig

```diff
@@ -3474,6 +3474,9 @@ pub const Type = extern union {
     pub fn structFieldOffset(ty: Type, index: usize, target: Target) u64 {
         switch (ty.tag()) {
             .@"struct" => {
+                if (index == 0) {
```

Comment:
I misunderstood what this function does, I think it would be better to change `abiSize()` directly